docs/app/docs/ide_configuration/direnv.md

```diff
@@ -0,0 +1,43 @@
+---
+title: direnv configuration
+---
+
+
+## direnv
+___
+[direnv](https://direnv.net) is an open source environment management tool that allows setting unique environment variables per directory in your file system. This guide covers how to configure direnv to seemlessly work with a devbox project.
+
+### Prerequisites
+* Install direnv and hook it to your shell. Follow [this guide](https://direnv.net/#basic-installation) if you haven't done it. 
+
+### Setting up with Devbox Shell and direnv
+
+Note: If you already have a devbox project you may skip to step 3.
+
+1. `devbox init` if you don't have a devbox.json in the root directory of your project.
+2. `devbox shell -- 'ls'` to activate devbox shell temporarily and make sure dependencies mentioned in your devbox.json are installed.
+3. Create a new file, name it `.envrc` and put the following snippet inside it:
+    ```bash
+    use_devbox() {
+        watch_file devbox.json
+        eval $(devbox setup-direnv)
+    }
+    use devbox
```

Comment:
I think this should have an underscore: `use_devbox` ?

Reply:
No. Strangely, no need for underscore 😄